src/test/java/nl/dtls/fairdatapoint/service/impl/FairMetaDataServiceImplTest.java

```diff
@@ -74,8 +74,8 @@
 @DirtiesContext
 public class FairMetaDataServiceImplTest {
 
-    private final static Logger LOGGER
-            = LogManager.getLogger(FairMetaDataServiceImplTest.class.getName());
+    private final static Logger LOGGER = LoggerFactory.getLogger(
+            FairMetaDataServiceImplTest.class.getName());
```

Comment:
We can use the class itself instead of resolving it per name. (also for consistency)